draft-frindell-webtrans-devious-baton.md

```diff
@@ -126,12 +133,12 @@ closes it.
 When either endpoint receives a Baton message on a stream, it takes the
 following actions:
 
-* If the value of the baton is 0, the endpoint MUST close the WebTransport
-  session with no error
+* If the value of the baton is 0, the endpoint decrements the number of active
+  batons by one.
 * If the value of the baton is not 0, the endpoint MUST send a new Baton message
   with a baton value equal to the incoming baton value + 1 modulo 256.  The new
   Baton message is sent on a stream, decribed below.
-* After sending the Baton message, the endpoint MUST close the stream
+* After sending the Baton message, the endpoint MUST close the stream.
 
```

Comment:
That's not 100% true. or rather it depends what you mean by "close". Sending the FIN bit, yes, always. But if the stream is a BIDIR stream opened by the local node, it is only closed in the sending direction, since the other direction will be used to receive the next relay by the peer.